src/certs.py

```diff
@@ -0,0 +1,89 @@
+"""Helpers for generating certificates."""
+import tempfile
+from pathlib import Path
+from subprocess import check_call
+
+SSL_CONFIG_FILE = "src/templates/ssl.conf.j2"
+
+
+def gen_certs(model: str, service_name: str):
+    """Generate certificates."""
+    # TODO: Refactor this into a python-based method that can be imported from Chisme
+    # generate SSL configuration based on template
+
+    ssl_conf = Path(SSL_CONFIG_FILE).read_text()
+    ssl_conf = ssl_conf.replace("{{ model }}", str(model))
+    ssl_conf = ssl_conf.replace("{{ service_name }}", str(service_name))
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        Path(tmp_dir + "/seldon-cert-gen-ssl.conf").write_text(ssl_conf)
```

Comment:
Just a question. Why we have have the name as `seldon-*`? Is it related to Seldon somehow?

Reply:
Nah, this is because I copied and pasted this from your charm and missed the reference :)  I'll fix it

Reply:
pending CI, this should be fixed by [this commit](https://github.com/canonical/namespace-node-affinity-operator/pull/3/commits/e7f79c9a1c1f0893fa97c4f518143355d627dce6)